### DISCUSSION TITLE
Problem to open Stepmania 5.1 Beta 2 (Problema al abrir Stepmania 5.1 Beta 2)

### DISCUSSION THREAD

**Discussion by Samirachuni** [stepmania/stepmania#2177] [Q&A]
Español:
Hey hola!! quisiera comentar un problema que tengo al abrir Stepmania 5.1 Beta 2. Hace tiempo que formatearon mi pc, y allí quería redescargarme el StepMania 5.1 Beta 2 en 2 pcs, la mia y la de mi hermana pequeña, el problema es que hay un error el cual quiero solucionar pero no se como, he buscado tutoriales de como solucionarlo pero no habia ninguno, si sabes la solucion, responde a esta discusión o hablamos en mi discord: Samirachuni#1824
PD: no se hablar muy bien ingles asi que usare el traductor de google xd

English:
Hey Hello!! I would like to comment on a problem I have when opening Stepmania 5.1 Beta 2. Some time ago my PC was formatted, and there I wanted to redownload StepMania 5.1 Beta 2 on 2 PCs, mine and my little sister's, the problem is that there is an error Which one do I want to solve but I don't know how, I have looked for tutorials on how to solve it but there weren't any, if you know the solution, reply to this discussion or we can talk on my discord: Samirachuni#1824
PS: I don't know how to speak English very well so I'll use the google translate xd

Imagen adjunta (Attached image):
![image](https://user-images.githubusercontent.com/97851043/149675193-4f6e42ce-0407-407b-9901-86c056f12f26.png)
![image](https://user-images.githubusercontent.com/97851043/149675215-99102388-ce65-4840-bffb-1304e42e0375.png)

Thanks for reading !!
Gracias por leer !!

**Comment by nico-abram** [comment]
Buenas @Samirachuni 
Me parece que ese error es el mismo que https://github.com/stepmania/stepmania/issues/1834
